<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>911</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CaLineKin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>181</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_basic_parameters">
     <attribute name="title">
      <string>Basic parameters</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pixel width (um)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Lines per second</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Upstroke threshold (0-1)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_basic_parameters_extracellular_space_threshold">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="toolTip">
       <string comment="&quot;&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This can be used to omit extracellular space from analysis. &quot; \                       &quot;This should be done (or you can remove the extracellular space as a pre-processing step yourself) or the analysis may be unreliable in the extracellular region. &quot; \                       &quot;Pixels with intensity below this threshold are omitted from the analysis. &quot; \                       &quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;&quot;" extracomment="&quot;&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This can be used to omit extracellular space from analysis. &quot; \                       &quot;This should be done (or you can remove the extracellular space as a pre-processing step yourself) or the analysis may be unreliable in the extracellular region. &quot; \                       &quot;Pixels with intensity below this threshold are omitted from the analysis. &quot; \                       &quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;&quot;"/>
      </property>
      <property name="whatsThis">
       <string comment="&quot;&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This can be used to omit extracellular space from analysis. &quot; \                       &quot;This should be done (or you can remove the extracellular space as a pre-processing step yourself) or the analysis may be unreliable in the extracellular region. &quot; \                       &quot;Pixels with intensity below this threshold are omitted from the analysis. &quot; \                       &quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;&quot;" extracomment="&quot;&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This can be used to omit extracellular space from analysis. &quot; \                       &quot;This should be done (or you can remove the extracellular space as a pre-processing step yourself) or the analysis may be unreliable in the extracellular region. &quot; \                       &quot;Pixels with intensity below this threshold are omitted from the analysis. &quot; \                       &quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;&quot;"/>
      </property>
      <property name="text">
       <string>Extracellular space threshold (fl. units)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_parameters_basic_pixel_width">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.14</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_parameters_basic_fps">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>416</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_parameters_upstroke_threshold">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_parameters_extracellular_space_threshold">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_advanced_parameters">
     <attribute name="title">
      <string>Advanced parameters</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Temporal zoom to attain this interval (ms)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="preprocessing_lineEdit_temporal_zoom">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>1D Gauss filter width (um)</string>
      </property>
     </widget>
     <widget class="QLabel" name="ap_image_preprocessing_label_median_duration_pixels">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Smooth to get peaks</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="preprocessing_lineEdit_SG_width">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>13</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="preprocessing_lineEdit_1dGaussFilter">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Savitzy Gloay temporal filtering width</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="preprocessing_lineEdit_smooth_peaks">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>50</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Output Stats</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Main analysis</string>
    </property>
    <widget class="QPushButton" name="button_analyze_data">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>87</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Analyze</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_load_image">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_crop">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>54</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Crop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_save_tiff">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save Tiffs</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Select Transient Parameters</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bottom Crop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>Top Crop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Left Crop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Right Crop</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_parameters_crop_left">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1024</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_parameters_crop_right">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1024</number>
     </property>
     <property name="value">
      <number>512</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_parameters_crop_bottom">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_parameters_crop_top">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>351</width>
      <height>841</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>461</width>
      <height>441</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>459</width>
       <height>439</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>431</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
